src/data-workspace/data-entry-cell/entry-field-input.js

```diff
@@ -92,27 +88,22 @@ export function EntryFieldInput({
         [setRightHandPanel]
     )
 
+    // todo: inner wrapper?
     const onFocus = useCallback(() => {
         setHighlightedFieldId({ de, coc })
     }, [de, coc, setHighlightedFieldId])
 
     const sharedProps = useMemo(
         () => ({
             fieldname,
-            dataValueParams,
+            deId: de.id,
+            cocId: coc.id,
             disabled,
             setSyncStatus,
             onFocus,
             onKeyDown,
         }),
-        [
-            fieldname,
-            dataValueParams,
-            disabled,
-            setSyncStatus,
-            onFocus,
-            onKeyDown,
-        ]
+        [fieldname, de, coc, disabled, setSyncStatus, onFocus, onKeyDown]
```

Comment:
dataValueParams isn't needed at the input-level anymore since the mutation functions handle it, so now just passing `deId` and `cocId` to the input components